cliquet/initialization.py

```diff
@@ -373,15 +374,45 @@ def on_new_response(event):
     config.add_subscriber(on_new_response, NewResponse)
 
 
+def _filter_events(callback, actions, resources):
+    def wrapped(event):
+        action = event.payload.get('action')
+        resource_name = event.payload.get('resource_name')
+        if action and action not in actions:
+            return
+        if resource_name and resources and resource_name not in resources:
+            return
+        return callback(event)
```

Comment:
I am not sure to understand what _filter_events does. It seems to : 
- call the callback and return the result
- not call it and return implicitely None

But then in the code it;s used like this:

```python
callback = _filter_events(callback, actions, resources)
```
So I am confused about what is "callback" exactly. I think this can be clarified by:

- a better naming
- explicit returns everywhere in this filtering function